"Security violation: unexpected connection attempt by "	,	lr_2
getInetAddress	,	vr_11
server	,	vr_1
disconnect	,	fn_11
getLocalPort	,	vr_7
verifyRemote	,	fn_4
OutputStream	,	tp_4
getCharset	,	fn_10
getLocalAddress	,	fn_2
getBytes	,	fn_3
NULL_CHAR	,	vl_1
password	,	vr_14
separateErrorStream	,	vr_16
getErrorStream	,	fn_8
flush	,	vr_8
_errorStream_	,	vl_4
enable	,	vr_20
getHostAddress	,	fn_5
buffer	,	vr_19
write	,	vr_6
close	,	vr_10
_output_	,	vr_5
read	,	vr_18
isRemoteVerificationEnabled	,	fn_13
ch	,	vr_17
InputStream	,	tp_1
createErrorStream	,	fn_1
"Server closed connection."	,	lr_3
ServerSocket	,	tp_2
SocketInputStream	,	fn_6
_input_	,	vl_3
_serverSocketFactory_	,	vr_3
getInputStream	,	vr_12
command	,	vr_15
accept	,	vr_9
"UTF-8"	,	lr_1
setRemoteVerificationEnabled	,	fn_12
rexec	,	fn_9
getOutputStream	,	fn_7
socket	,	vr_2
remoteVerificationEnabled	,	vl_2
Socket	,	tp_3
createServerSocket	,	vr_4
username	,	vr_13
